src/components/PageSection.tsx

```diff
@@ -5,15 +5,16 @@ type Props = {
   description: string
   position: string
   image: string
+  class?: string
```

Comment:
whats class? seems like it can only be 'EVENTS' or undefined? is there other use cases

Reply:
yeah, can only be events, but for future proofing as there might be different styling for this component in the future.